app/javascript/packs/pw-strength.js

```diff
@@ -96,6 +96,17 @@ function disableSubmit(submitEl, length = 0, score = 0) {
   }
 }
 
+/**
+ * @param {HTMLElement?} element
+ *
+ * @return {string[]|undefined}
+ */
+export function getForbiddenPasswords(element) {
+  try {
+    return JSON.parse(element.dataset.forbiddenPasswords);
+  } catch {}
```

Comment:
what if we returned empty array in the catch case? so the function always returns an array?

```suggestion
  } catch {
    return [];
  }
```

Reply:
> what if we returned empty array in the catch case? so the function always returns an array?

In my mind the thinking was to try to be a bit more intentional about passing `undefined` as in letting the default behavior of an absent second argument to `zxcvbn` take effect. But now that you mention it, it could make things simpler to have just one return type, especially since it should have the same meaning, and can avoid needing to reconfigure the edited ESLint rule (though I don't like the default rule regardless 😅).

Reply:
Yeah as a rule-of-thumb I think having as simple of a return type as possible usually benefits the calling code

Reply:
> what if we returned empty array in the catch case? so the function always returns an array?

Updated in cdafa30.

Kept changes from 240818d, but LMK if you have a preference on this.

Reply:
> Kept changes from [240818d](https://github.com/18F/identity-idp/commit/240818d6275b0a03d50667813f8401d0c0e2af1e), but LMK if you have a preference on this.

I don't have strong feelings about the ESLint rules, since I don't know them that well, or run into them very often when working on the IDP, happy to trust your changes to them